modules/ROOT/pages/logging-mdc.adoc

```diff
@@ -4,74 +4,98 @@ include::_attributes.adoc[]
+. Replace `[processor: %X{processorPath}; event: %X{correlationId}]` with `[%MDC]`.
 
+=== Example log4j.xml File Configurations
+
+.Example: Default log4j.xml file without MDC logging.
 ----
 <PatternLayout pattern="%-5p %d [%t] [processor: %X{processorPath}; event: %X{correlationId}] %c: %m%n"/>
 ----
 
-To this:
-
+.Example: Updated log4j.xml file with MDC logging.
 ----
 <PatternLayout pattern="%-5p %d [%t] [%MDC] %c: %m%n"/>
 ----
 
-Changing `[processor: %X{processorPath}; event: %X{correlationId}]` to `[%MDC]`. This will automatically add the MDC context (which includes the correlation id as well as the processor path).
-
-==== Available operations
+== Available Operations
 
 // This will probably need to be set as a table as in other extensions
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+|XML Element | Description | Accepted Attributes
+|`set-logging-variable` | Sets a logging variable and its value. |
+[%header%autowidth.spreadcols="a,a"]
+!===
+! Attribute Name ! Description
+!`variableName` ! The name of the variable to log.
+! `value` ! The value to assign to the variable. Accepts DataWeave expressions.
+!===
+| `remove-logging-variable` | Removes a logging variable. |
+[%header%autowidth.spreadcols="a,a"]
+!===
+! Attribute Name ! Description
+! `variableName` ! The name of the variable to remove.
+!===
+| `clear-logging-variables` | Removes all the logging variables. This option does not remove the processor path nor the correlation ID
+| None
+|===
+
+== Configure MDC Logging in Your Application
+
+To configure MDC logging, add any of the available operations to your Mule application flow and specify the corresponding attributes.
+
+For example, to configure a logging variable, insert the `<tracing:set-logging-variable>` XML element into your application flow:
+----
+<flow name="exmapleFlow">
+    ...
+    <tracing:set-logging-variable variableName="testVar" value="testValue" />
+    ...
+</flow>
+----
+
+After executing the flow, the output logs are:
+----
+INFO  2021-04-08 16:58:26,882 [[MuleRuntime].uber.15: [test-project-app].exmapleFlow.CPU_LITE @18f679] [{correlationId=c85e16c0-98a4-11eb-bc34-cac765a2219b, processorPath=exmapleFlow/processors/2, testVar=testValue}] org.mule.runtime.core.internal.processor.LoggerMessageProcessor: Example
+----
 
-* `set-logging-variable`: Sets the value to a logging variable. Attributes: `variableName` and `value` (accepts expression).
-* `remove-logging-variable`: Removes a logging variable. Attribute: `variableName`.
-* `clear-logging-variables`: removes all the logging variables. This will not remove the processor path or correlation id. Attributes: none.
+This logging context affects any output of a Logger Message Processor and any internal logging that Mule runtime produces. Therefore, you can add more context to the event that Mule is processing in case you need additional information for tracking down any potential issue, for example, an unexpected error occurred.
```

Comment:
`LoggerMessageProcessor` relates to the Logger component. You see it in the Studio console, for ex:  `org.mule.runtime.core.internal.processor.LoggerMessageProcessor : "hello"` I think we usually talk about the output of the Logger component instead of referring to a `LoggerMessageProcessor`  message.